builtin/providers/google/resource_compute_instance_template_test.go

```diff
@@ -115,6 +116,30 @@ func TestAccComputeInstanceTemplate_subnet_custom(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstanceTemplate_subnet_xpn(t *testing.T) {
+	var instanceTemplate compute.InstanceTemplate
+	var xpn_host = os.Getenv("GOOGLE_XPN_HOST_PROJECT")
+	if xpn_host == "" {
```

Comment:
If you omit this check, then the travis build will pass - we have this already setup with the correct parameter